nautobot/extras/tests/test_jobs.py

```diff
@@ -29,6 +40,15 @@ class JobTest(TestCase):
     def setUpTestData(cls):
         cls.job_content_type = ContentType.objects.get(app_label="extras", model="job")
 
+    def setUp(self):
+        super().setUp()
+        self.user = User.objects.create(username="Super User", is_active=True, is_superuser=True)
```

Comment:
Note that the base class (`nautobot.utilities.testing.views.TestCase`) already creates a `self.user`, though it won't be a superuser by default (do we need that for this test?), you can influence which permissions it has by defining a `user_permissions` tuple on the class.

Reply:
Ah, I was following another example I found in tests, I will try the self.user, I really just need a user that has some permissions. Not necessarily SuperUser.